lib/android/app/src/main/java/com/reactnativenavigation/options/parsers/ColorParser.java

```diff
@@ -0,0 +1,32 @@
+package com.reactnativenavigation.options.parsers;
+
+import android.content.Context;
+
+import com.facebook.react.bridge.ColorPropConverter;
+import com.reactnativenavigation.options.params.Colour;
+import com.reactnativenavigation.options.params.DontApplyColour;
+import com.reactnativenavigation.options.params.NullColor;
+
+import org.json.JSONObject;
+
+public class ColorParser {
```

Comment:
this file should be removed, since it is removed in main branch

Reply:
But it's being used in `ThemeColour.kt`, should it be replaced by something else?

Reply:
It's being used in `ThemeColour.kt` but I think it's being taken from `reactNative62` or somewhere else

Reply:
`import com.reactnativenavigation.options.parsers.ColorParser` import from original file

Reply:
In the main branch today it exists in a Kotlin format, but it's under the `reactNative63`, which was removed here, we need to replace it, I'm adding a note here just to remember to do that before merging,`ib/android/app/src/reactNative63/java/com/reactnativenavigation/options/parsers/ColorParser.kt`.

Reply:
I'll change it

Reply:
Done